tools/export/gcc_arm_nrf51_dk.tmpl

```diff
@@ -10,5 +14,5 @@ SREC_CAT = srec_cat
 
 {% block additional_targets %}
 merge:
```

Comment:
At minimum this should depend on $(Project).hex . Sorry I missed this when I unified the gcc_arm exporter.